charts/testkube/values.yaml

```diff
@@ -43,7 +43,11 @@ mongodb:
 
 testkube-api:
   prometheus:
-    enabled: false
+    enabled: true
```

Comment:
should it be true by default?

Reply:
no, forgot to remove it after testing. good catch!